### DISCUSSION TITLE
Attributes for Product Options and Product Option Values

### DISCUSSION THREAD

**Discussion by charlielangridge** [lunarphp/lunar#1296] [Roadmap]
So... product options and product option values - is anyone working on / is there any reason why there is only a "name" and not attribute_data as with product - and a 'handle' on values specifically - if there's no reason why not, i'm happy to convert them to have attribute_date, so as to be more extendable (as an example, we'd like to store paper sizes, so the label might be 'a6', the name 'ISO A6' and also store a 'size' attribute with '105x148mm'

**Comment by netzknecht** [comment]
In principle it is possible to [extend the ProductOption or ProductOptionValue models](https://docs-next.lunarphp.io/core/extending/models.html) and [adding attributes](https://docs-next.lunarphp.io/core/reference/attributes.html#adding-attributes-to-your-own-model) to them. But if you use the lunar hub, further adjustments are required.
